migrate/migrations/index.ts

```diff
@@ -41,4 +42,5 @@ export default {
   n18_logsFields,
   n19_connectionsUserinfo,
   n20_missingFields,
+  n21_missingFields,
```

Comment:
:eyes: this should be named differently

```suggestion
  n21_sessionDeletedAt,
```

Reply:
Ouch.. missed that one